Cargo.toml

```diff
@@ -80,12 +80,13 @@ mac-process-info = "0.2.0"
 osascript = "0.3.0"
 unix-daemonize = "0.1.2"
 
-[target."cfg(unix)"]
+#[target.'cfg(target_family = "unix")']
 
-[target."cfg(unix)".dependencies]
+[target.'cfg(target_family = "unix")'.dependencies]
```

Comment:
Do we need to be that explicit with `target_family` here instead of using just `cfg(unix)`?

Reply:
it's not "necessary". I simply sticked to the reference [doc](https://doc.rust-lang.org/reference/conditional-compilation.html#target_family).

If you like the _shortcut_ syntax more we can revert that. Personally though I like to be explicit when writing things. It's a bit more verbose but for people reading it, it might be easier to get what's going on.

Reply:
Usually, the rust convention is to use the shorter form, so I'd prefer to leave it as it was

Reply:
👍🏻